tests/ui/rfcs/rfc-2632-const-trait-impl/effects/const_closure-const_trait_impl-ice-113381.rs

```diff
@@ -1,3 +1,9 @@
+// known-bug: unknown
```

Comment:
```suggestion
// known-bug: 110395
```

Just change this to #110395 or add `FIXME(effects)`, since I'd like to grep them..